cmd/bbolt/main.go

```diff
@@ -1431,14 +1438,13 @@ func (cmd *BenchCommand) runWritesRandomNested(db *bolt.DB, options *BenchOption
 }
 
 func (cmd *BenchCommand) runWritesWithSource(db *bolt.DB, options *BenchOptions, results *BenchResults, keySource func() uint32) error {
-	results.WriteOps = options.Iterations
 
-	for i := 0; i < options.Iterations; i += options.BatchSize {
+	for i := int64(0); i < options.Iterations; i += options.BatchSize {
```

Comment:
Could we add a comment here on why the need for the conversion, I believe you mentioned due to int limitations, correct?